src/parser.rs

```diff
@@ -291,7 +301,15 @@ where
         match self.peek() {
             Some(TOKEN_DYNAMIC_START) => self.parse_dynamic(),
             Some(TOKEN_STRING_START) => self.parse_string(),
-            _ => self.expect_ident(),
+            Some(TOKEN_IDENT) => self.expect_ident(),
+            _ => {
+                let start = self.start_error_node();
+                let end = self.finish_error_node();
+                self.errors.push(ParseError::Expected(
+                    TextRange::new(start, end),
+                    "attribute".to_string(),
```

Comment:
I guess we should reflect here that strings etc. are also OK.

Reply:
What should be the exact message used? Maybe `attribute or string`? Although I guess a string does represent an attribute here? Please correct me if I'm wrong